### ISSUE TITLE
[BUG] color styles on th doesn't work ?

### ISSUE THREAD

**Issue by launtony** [PHPOffice/PhpSpreadsheet#451]
### Prerequisites

* [x] Able to reproduce the behaviour outside of your code, the problem is isolated to Laravel Excel.
* [ ] Checked that your issue isn't already filed.
* [ ] Checked if no PR was submitted that fixes this problem.

### Versions

* PHP version: 7.0
* Laravel version: 5.5
* Package version: 1.2

### Description

This code generate a cell with a black background and the text too.
```
<th style="background-color : #000000; color : #FFFFFF">{{ __('excel.firstColumn') }}</th>
```

### Steps to Reproduce

```
<th style="background-color : #000000; color : #FFFFFF">{{ __('excel.firstColumn') }}</th>
```

**Expected behavior:**

I need white color for the text :)

Line in the code : https://github.com/PHPOffice/PhpSpreadsheet/blob/5f036591cdc1a4f848edd83db8666bcf20874c78/src/PhpSpreadsheet/Reader/Html.php#L625

**Actual behavior:** 

I have black color with Excel for macOS and Windows. Only Preview and Numbers on macOS works

**Comment by launtony** [comment]
I have make some tests. Xls works fine but Xlsx doesn't work.

**Comment by launtony** [comment]
Has anyone ever had this problem?

**Comment by demobiel** [comment]
I have the same issue, it works in numbers and in preview, but if you open it in Excel it shows black text

**Comment by PowerKiKi** [comment]
Please provide a [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve) of code that exhibits this issue without relying on an external Excel file, if you want to increase the chances of getting help on this.

**Comment by oleibman** [comment]
Reported fixed in PhpSpreadsheet 1.6 in issue #838.
